lib/memory-store.js

```diff
@@ -52,7 +52,8 @@ lease_store.prototype.get_lease_by_ip = function get_lease_by_ip (ip, cb) {
   cb(lease);
 };
 lease_store.prototype.get_next_ip = function get_next_ip (cb) {
-  var latest = Object.keys(ip_map).sort()[-1];
+  var leasedIPs = Object.keys(ip_map);
+  var latest = leasedIPs.sort()[leasedIPs.length - 1];
```

Comment:
-1 should work just fine as an array index. Is this just splitting out for understandability?